.github/workflows/formatting.yml

```diff
@@ -0,0 +1,103 @@
+name: formatting-check
+
+on:
+  push:
```

Comment:
This enables running the checks on pushes to private forks which I kinda like. No need to create local pull requests to have formatting checked.